generateLJFF.py

```diff
@@ -3,39 +3,55 @@
 import sys
 
 import __main__ as main
-import numpy as np
 
-import ppafm as PPU
-import ppafm.fieldFFT as fFFT
-import ppafm.GridUtils as GU
-import ppafm.HighLevel as PPH
-from ppafm import basUtils, elements
+import ppafm
+from ppafm.HighLevel import computeLJ
 
-if __name__=="__main__":
-    HELP_MSG="""Use this program in the following way:
+if __name__ == "__main__":
+    HELP_MSG = """Use this program in the following way:
     %s -i <filename>
 
     Supported file fromats are:
        * xyz
-    """ %os.path.basename(main.__file__)
```

Comment:
Why does it divide this into multiple lines?

Reply:
I don't know this. Probably black considers a docstring as a single line, no idea.